config.yaml

```diff
@@ -38,25 +38,29 @@ menu:
       url: https://blog.gitea.io/
       weight: 30
       pre: rss
+    - name: Instances
+      url: /instances
+      weight: 40
+      pre: list
     - name: Code
       url: https://code.gitea.io/
-      weight: 40
+      weight: 50
```

Comment:
There is no need to modify the weight of the other items in the list. You can set instances weight to 35 and achieve the same outcome.